packages/core/src/asset/request.ts

```
@@ -54,16 +54,21 @@
     config.method = config.method ?? "get";
-
+    // @ts-ignore
+    xhr.responseType = isImg ? "blob" : config.type;
     xhr.onload = () => {
       if (xhr.status < 200 || xhr.status >= 300) {
         reject(new Error(`request failed from: ${url}`));
         return;
       }
-      const result = xhr.response ?? xhr.responseText;
+      const responseType = xhr.responseType;
+      let result = (responseType == '' || responseType == 'text') ? xhr.responseText : xhr.response;
```

ESLint

[error] Replace `(responseType·==·''·||·responseType·==·'text')` with `responseType·==·""·||·responseType·==·"text"`

(prettier/prettier)